api/v1beta1/common_types.go

```diff
@@ -57,6 +57,14 @@ const (
 	// update that disallows a pre-existing Cluster to be populated with Topology information and Class.
 	ClusterTopologyUnsafeUpdateClassNameAnnotation = "unsafe.topology.cluster.x-k8s.io/disable-update-class-name-check"
 
+	// MachinePoolOwnedLabel is the label set on Machines and InfraMachines linked to a MachinePool.
+	MachinePoolOwnedLabel = "machinepool.cluster.x-k8s.io/owned"
+
+	// FallbackMachineLabel indicates that a Machine belongs to a MachinePool that does not support MachinePool Machines.
+	// As such, these Machines exist to create a consistent user experience and will not have an infrastructure reference. The user will
+	// also be prevented from deleting these Machines.
+	FallbackMachineLabel = "machinepool.cluster.x-k8s.io/fallback-machine"
```

Comment:
Yeah, I'm open to coming up with a better name. The MPM proposal described these as "CAPI-only" Machines which seemed a bit confusing so I tried to rename it. Do you have any ideas?

Reply:
I don't think we should have a different name. 
It is just a regular machine without the capability to trigger deletion.

Reply:
Just to clarify, do you mean that we should keep the name "fallback machines" or rename it to "CAPI-only" in the code?

Reply:
> Just to clarify, do you mean that we should keep the name "fallback machines" or rename it to "CAPI-only" in the code?

I don't think we should have a different name.
It is just a regular machine without the capability to trigger deletion, and this can be modeled e.g. with a new field in status (optional, defined by the contract), signaling that the Machine has limited capabilities.
cc @vincepri

Reply:
I think fallback-machine is not a good option. I'm not sure about status.

If we put "not deletable" in there it wouldn't be exactly true as those machines are just not deleteable by the user, right? (but they are deletable by the controller)

I wonder if we basically have the same concept as "mirror pods" that kubelet is using for static pods (kube-apiserver, ...). The UX there is that you can delete them but nothing happens (which is not great)

Reply:
I like the idea of "static machine"

`StaticMachineLabel = "machinepool.cluster.x-k8s.io/static-machine"`

thoughts?